apps/passport-client/components/screens/ScanScreen.tsx

```diff
@@ -41,37 +62,9 @@ export function ScanScreen(): JSX.Element {
             <Back />
             <Home />
           </ButtonsContainer>
-          <QrReader
-            className="qr"
-            onResult={(result, error): void => {
-              if (result) {
-                console.log(
-                  `Got result, considering redirect`,
-                  result.getText()
-                );
-                const newLoc = maybeRedirect(result.getText());
-                if (newLoc) {
-                  // Instantly remove any error toasts
-                  toast.remove();
-                  nav(newLoc);
-                } else {
-                  toast.error(
-                    "The QR code you scanned is not a Zupass QR code. Make sure the QR code you're scanning comes from the Zupass app.",
-                    {
-                      id: "scan-error",
-                      duration: 10000,
-                      position: "bottom-center"
-                    }
-                  );
-                }
-              } else if (error) {
-                //    console.info(error);
-              }
-            }}
-            constraints={{ facingMode: "environment", aspectRatio: 1 }}
-            ViewFinder={ViewFinder}
-            containerStyle={{ width: "100%" }}
-          />
+          <Spacer h={8} />
+          {sdkState === "initialized" && <StrichScanner onResult={onResult} />}
```

Comment:
potentially worth rendering something in the case `sdkState` is `undefined` - 'initializing scanner...'